### DISCUSSION TITLE
Proposal: Use Mantine UI

### DISCUSSION THREAD

**Discussion by cr101** [epicweb-dev/epic-stack#88] [Ideas]
I love EpicStack and I was wondering if the stack could make it easier somehow to swap Radix UI with [Mantine UI](https://mantine.dev/) . While Radix UI is great for small projects, Mantine is much more useful for building a SaaS application as it's an extensive component library with over 100 fully styled components ready-to-use.

**Comment by kentcdodds** [comment]
You definitely can use Mantine if you like. But I'm not going to add it as a built-in to the template because it uses CSS-in-JS which cannot support streaming which we need for important features like [`defer`](https://remix.run/docs/en/1.16.1/guides/streaming#using-defer).

**Reply by cr101** [reply]
[Mantine v7](https://v7.mantine.dev/changelog/7-0-0) is moving from CSS-in-JS to native CSS
